### DISCUSSION TITLE
which CSS framework to use

### DISCUSSION THREAD

**Discussion by issa-hassane** [Sunbird-RC/vc-issuance#17] [General]
is it ok to use react with tailwind css ??

**Comment by jayadinni** [comment]
Please use this design system: 


<a href="https://www.figma.com/file/j0Rl8R0xVKMko2EzxPW3rE/Sunbird-RC-Design-System?node-id=59%3A296" target="_blank"><img src="https://user-images.githubusercontent.com/580711/194706290-b281e460-df87-4bf2-a83e-af3b718e1fc8.png" width=600></a>
